pulp-glue/pulp_glue/rpm/context.py

```diff
@@ -65,8 +65,14 @@ def preprocess_entity(self, body: EntityDefinition, partial: bool = False) -> En
             if "repository" not in body and "publication" in body:
                 body["repository"] = None
         if body.get("generate_repo_config") is False:
-            self.pulp_ctx.needs_plugin(PluginRequirement("rpm", specifier=">=3.23.0.dev"))
-        elif self.pulp_ctx.has_plugin(PluginRequirement("rpm", specifier="<3.23.0.dev")):
+            self.pulp_ctx.needs_plugin(
+                PluginRequirement(
+                    "rpm",
+                    specifier=">=3.23.0",
+                    feature=_("Generation of config.repo file."),
```

Comment:
```suggestion
                    feature=_("configuring the generation of a config.repo file"),
```
It's really embedded in another sentence.